code/mob/living/critter/small_animal.dm

```diff
@@ -1484,6 +1487,18 @@ var/list/mob_bird_species = list("smallowl" = /mob/living/critter/small_animal/b
 				return 2
 		return ..()
 
+	attack_hand(mob/living/M)
+		if (ishuman(M) && M.a_intent == INTENT_HARM && prob(95))
```

Comment:
```suggestion
		if (ishuman(M) && M.a_intent == INTENT_HARM)
```
`prob(95)` is dumb unless there's some joke I'm missing
If you want to keep it at least add a "THE ROACH RESISTS YOUR STOMP!" message and stun them or something on that 5%

Reply:
5% chance of the roach desperatly hanging onto life.